workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/Templates/UnityComponentDataGenerator.tt

```diff
@@ -76,6 +76,20 @@ namespace <#= qualifiedNamespace #>
 <# } #>
         }
 
+        public struct Snapshot : ISpatialComponentSnapshot
+        {
+            public uint ComponentId => <#= componentDetails.ComponentId #>;
+
+<# foreach(var fieldDetails in fieldDetailsList) { #>
+<# if (fieldDetails.IsBlittable) { #>
```

Comment:
This if-else doesn't appear to be necessary

Reply:
Yep. Originally had them different. Didn't notice they are now the same.